nameko/messaging.py

```diff
@@ -388,34 +388,6 @@ def on_consume_ready(self, connection, channel, consumers, **kwargs):
             else:
                 callback()
 
-    def consume(self, limit=None, timeout=None, safety_interval=0.1, **kwargs):
-        """ Lifted from Kombu.
-
-        We switch the order of the `break` and `self.on_iteration()` to
```

Comment:
is this no longer an issue? how long is that timeout and would stopping have to wait for it without this?